.github/workflows/test.yml

```diff
@@ -341,7 +341,12 @@ jobs:
       with:
         template: templates/vmnet.yaml
     - name: Install test dependencies
-      run: brew install qemu bash coreutils iperf3
+      run: |
+        brew install bash coreutils iperf3
+        # Install qemu 8.2.1. Unfortunately this mechanism builds from source.
```

Comment:
What is the issue with the latest qemu?
Also, could you report the issue to https://gitlab.com/qemu-project/qemu/-/issues

Reply:
I have no idea; I spent almost 6 hours trying different things on both macOS 12 and 13, and then only time I got a passing upgrade test was when I downgraded qemu.

The issue is that Lima cannot establish an SSH connection, and then the runner somehow gets lost and the jobs seems skipped.

The issue seems specific to GitHub runners; the same setup works fine on my local machine.